ci/build_container/build_container_centos.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+
+yum update -y
+
+# scl devtoolset and epel repositories
+yum install -y centos-release-scl epel-release
+yum install  -y https://centos7.iuscommunity.org/ius-release.rpm
+
+# llvm-5.0.1 repository from copr
+curl -L -o /etc/yum.repos.d/alonid-llvm-5.0.1-epel-7.repo \
```

Comment:
If `clang` isn't supported, then this isn't necessary.

Reply:
I will either test the other build steps using clang and leave it there or remove this as you suggested.